lesson3/21_nil_receiver.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+type Line struct {
+	x1, y1 float64
+	x2, y2 float64
+}
+
+func (line *Line) length() float64 {
+	if line == nil {
+		return 0
+	}
+    return math.Hypot(line.x1-line.x2, line.y1-line.y2)
```

Comment:
pls. reformat

Reply:
Reformatted via go fmt. If you meant adding an empty line after the ```if``` block, it will not fit the slide.

Reply:
no, I mean just those 4spaces => tab thingy. TYVM